__tests__/app.test.js

```diff
@@ -183,6 +183,49 @@ describe("GET /api/articles/:articleid/comments", () => {
         .get("/api/articles/northcoders/comments")
         .expect(400)
         .then( ( { body: { msg } } ) => {
-            expect(msg).toBe("Invalid article_id")
+            expect(msg).toBe("Invalid database input")
         })})
-})
\ No newline at end of file
+})
+
+describe("PATCH /api/articles/:article_id", () => {
+    test("200: responds with updated article", () => {
+        return request(app)
+        .patch("/api/articles/1")
+        .send({ inc_votes: 1 })
+        .expect(200)
+        .then( ( { body: { article } } ) => {
+            console.log(article)
+            expect(article.votes).toBe(101)
+        })
+    })
+
+    test("400: inc_votes key not found", () => {
+        return request(app)
+        .patch("/api/articles/2")
+        .send({ votes: 1 })
+        .expect(400)
+        .then( ( { body: { msg } } ) => {
+            expect(msg).toBe("inc_votes key not found")
+        })
+    })
+
+    test("400: inc_votes is not a valid number", () => {
+        return request(app)
+        .patch("/api/articles/2")
+        .send({ inc_votes: "a" })
+        .expect(400)
+        .then( ( { body: { msg } } ) => {
+            expect(msg).toBe("Invalid database input")
+        })
+    })
+    test("400: object has more than one key", () => {
```

Comment:
This is fine here, alternatively you could test "200: additional unnecessary keys are ignored" instead, there's arguments for both sides :)